model/factories.go

```diff
@@ -0,0 +1,127 @@
+/*
+DESCRIPTION
+	factories.go provides a factory pattern to generate common
+	implementations of entities.
+
+AUTHORS
+	David Sutton <davidsutton@ausocean.org>
+
+LICENSE
+	Copyright (C) 2024 the Australian Ocean Lab (AusOcean).
+
+	This is free software: you can redistribute it and/or modify it
+	under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	This is distributed in the hope that it will be useful, but WITHOUT
+	ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+	or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+	License for more details.
+
+	You should have received a copy of the GNU General Public License
+	in gpl.txt. If not, see http://www.gnu.org/licenses/.
+*/
+
+package model
+
+import (
+	"github.com/ausocean/utils/nmea"
+)
+
+// Sensor factories //
+
+// Consts used by sensor factories.
+const (
+	tempLinearArgs      string = "0.1,-273.15"
```

Comment:
I reckon these should be const floats and then converted to strings when required.